tensorflow/contrib/learn/python/learn/datasets/mnist.py

```diff
@@ -28,7 +28,7 @@
 from tensorflow.python.framework import dtypes
 
 SOURCE_URL = 'http://yann.lecun.com/exdb/mnist/'
-
+BACKUP_URL = 'https://web.archive.org/web/20160828233817/http://yann.lecun.com/exdb/mnist/'
```

Comment:
Sure, that would be a simple modification. If anyone has a better canonical backup URL than `web.archive.com`, feel free to suggest it, too. I don't want that site to get overwhelmed...